uhini_newsletter/app.js

```diff
@@ -0,0 +1,61 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const request = require("request");
+const app = express();
+
+app.use(express.static("public"));
+app.use(bodyParser.urlencoded({extended: true}));
+app.get("/",function(req,res) {
+  res.sendFile(__dirname + "/signup.html");
+});
+
+app.post("/",function(req,res) {
+  var firstName = req.body.fname;
+  var lastName = req.body.lname;
+  var email = req.body.email;
+  var data = {
+    members :[
+      {
+        email_address: email,
+        status: "subscribed",
```

Comment:
It's a Node Application, and it does not follow the Project Tech Stacks 
So I am unable to allow this to merge